deploy/resources/knative-serving-0.6.0.yaml

```diff
@@ -1656,3 +1656,10 @@ spec:
         name: config-logging
 
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    serving.knative.dev/release: "v0.6.0"
+  name: config-service-ca
+  namespace: knative-serving
```

Comment:
I would like to keep this file as close to identical to upstream's 0.6.0 release as possible, especially when it comes to platform-specific features (openshift). You're already creating CM in the operator, so there should be no need for it to be here, right?

Reply:
Iam keeping this here because in transform function for all the files ownerReference will be added So if i create a configmap in install function then ownerReference "serving.knative.dev/v1alpha1" will not be added for newly created configmap...
So in the operator iam just updating configmap with annotation and checking the condition whether configmap exist or not

Reply:
I understand your logic, but we don't want the CM in this file. :)

This file should match the upstream release artifact. That's why `0.6.0` is in its name. The only time we might change something in here is if we need to cherry-pick some upstream fix prior to the next release.

Remember that this operator might be used to install KS on any number of different platforms (GKE, Azure, minikube, etc) but only OpenShift is going to do anything with a CM named `config-service-ca`, and that Azure user is going to wonder why it's there. :)

So I'm afraid you'll need to change `ensureConfigMapForCrt` back from a Transformer to a PreInstall hook. And you can set that owner reference as easy as you're setting that annotation:

    import "k8s.io/apimachinery/pkg/apis/meta/v1"
    func ensureConfigMapForCrt(instance *servingv1alpha1.Install) error {
        ...
        cm.SetOwnerReferences([]v1.OwnerReference{*v1.NewControllerRef(instance, instance.GroupVersionKind())})
        ...

Make sense?

Reply:
Okey thank you i understood the context i will change and update the code..